app/src/lib/helpers/linux.ts

```diff
@@ -82,9 +82,10 @@ export function spawnEditor(
   options: SpawnOptions
 ): ChildProcess {
   if (isFlatpakBuild()) {
+    let EscapedworkingDirectory = workingDirectory.replace(/(\s)/, "\ ")
```

Comment:
Are we able to extract this out to it's own function, and add a specific test or tests to `app/test/unit/helpers/linux-test.ts` so we can understand what it's resolving, and ensure we don't regress it in the future?

e.g.

```ts
export function formatWorkingDirectoryForFlatpak(path: string) {
   ...
}
```

Reply:
yeah it should be possible to that

Reply:
I've added the test.